classes/WP_CLI/Action/Get.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * WP-CLI command: action-scheduler action get
+ */
+class ActionScheduler_WPCLI_Action_Get_Command extends ActionScheduler_WPCLI_Command {
+
+	/**
+	 * Execute command.
+	 *
+	 * @return void
+	 */
+	public function execute() {
+		$action_id = $this->args[0];
+		$store     = \ActionScheduler::store();
+		$logger    = \ActionScheduler::logger();
+		$action    = $store->fetch_action( $action_id );
```

Comment:
Same as delete, this throws a fatal for an action that does not exist. Let's handle that gracefully like `wp post` does.